app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,17 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(self,
+                 nickname: str,
+                 favourite_dish: str,
+                 skill_level: int) -> None:
+        super().__init__(nickname, favourite_dish)

```

Comment:
would be better to use : `super().__init__(nickname=nickname, favourite_dish=favourite_dish)`